onnx/compose.py

```diff
@@ -76,7 +78,7 @@ def _edge_names(graph: GraphProto, exclude: Optional[Set[str]] = None) -> List[s
     return result
 
 
-def merge_graphs(
+def merge_graphs(  # pylint: disable=too-many-branches,too-many-statements
```

Comment:
Do you think it is OK for us to disable too-many-branches and too-many-statements globally?

Reply:
I think it is still a good reminder to write small(er) functions. We use this to exempt existing functions, but future functions, if having many branches or statements, should still declare their conscious intent by adding this disable inline.

Reply:
What's the cutoff size for triggering this?

Reply:
I think it is 50 for statements and 12 for branches. It can be adjusted